fix: sort entries before printing for fp-finder command

S0obi opened the pull request:
Seems like I refactored too much with initial PR #219 and inadvertently removed the sorting 😊
This commit will fix it.

theseion reviewed (changes requested):
Please add a test the ensures that entries are sorted.

S0obi commented:
> Please add a test the ensures that entries are sorted.

I refactored a bit the main function to make it easily testable. I added a test to be sure entries are sorted and duplicates have been removed.

theseion commented on util/fp_finder_test.go:
```diff
@@ -38,6 +38,20 @@ func (s *fpFinderTestSuite) TestFpFinder_FilterContent() {
 	s.Equal(expected, result)
 }
 
+func (s *fpFinderTestSuite) TestFpFinder_ProcessWords() {
```
Add another test that checks sorting explicitly.

theseion commented on util/fp_finder_test.go:
```diff
@@ -38,6 +38,31 @@ func (s *fpFinderTestSuite) TestFpFinder_FilterContent() {
+	input := []string{"pear", "Banana", ".hiddenfruit", "kiwi", "banana", "Apple", ".dotfruit"}
+	dict := map[string]struct{}{} // empty dictionary, so no filtering
+
+	expected := []string{".dotfruit", ".hiddenfruit", "Apple", "Banana", "banana", "kiwi", "pear"}
```
Matches should be case-insensitive.
